src/Workleap.AspNetCore.Authentication.ClientCredentialsGrant/AuthenticationBuilderExtensions.cs

```diff
@@ -42,6 +43,19 @@ public static AuthenticationBuilder AddClientCredentials(this AuthenticationBuil
             builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<IValidateOptions<JwtBearerOptions>>(new JwtBearerOptionsValidator(authScheme)));
             builder.Services.TryAddEnumerable(ServiceDescriptor.Singleton<IPostConfigureOptions<JwtBearerOptions>>(new ClientCredentialsPostConfigureOptions(authScheme)));
         }
+        
+        builder.Services.ConfigureSwaggerGen(options =>
```

Comment:
Should we use a PostConfiguration for this to put our OperationFilter at the end?